sdk/operationalinsights/Microsoft.Azure.OperationalInsights/src/Microsoft.Azure.OperationalInsights.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Microsoft Azure Operational Insights Data Library</Description>
+    <PackageId>Microsoft.Azure.OperationalInsights</PackageId>
+    <Description>Provides Microsoft Azure Operational Insights data operations</Description>
+    <AssemblyName>Microsoft.Azure.OperationalInsights</AssemblyName>
+    <VersionPrefix>0.10.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure OperationalInsights;OperationalInsights</PackageTags>    
+	  <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
+
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
```

Comment:
Similar to my comments in HDInsights PR we shouldn't be setting these but instead we should be fixing them to work with the data-plane targets.

Reply:
Ok, I am updating this now.